### ISSUE TITLE
sqlite constraint failed attachment

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-android#627]
when opening message

did not crash the app or otherwise cause trouble

```
/System.out: * 119 FETCH (BODY[2.MIME] {215}
I/System.out: Content-Type: application/octet-stream; name="0x0D5688EBF3102BE7.asc.pgp"
    Content-Disposition: attachment; filename="0x0D5688EBF3102BE7.asc.pgp"
    Content-Transfer-Encoding: base64
    X-Attachment-Id: f_UgPXIpPpLq
    )
I/System.out: B20 OK Success
E/SQLiteDatabase: Error inserting uid=1404 folder=[Gmail]/All Mail forwarded_uid=0 name=0x0D5688EBF3102BE7.asc.pgp attachment_id=f_UgPXIpPpLq encodedSize=2930 type=APPLICATION/OCTET-STREAM forwarded_folder=null email=flowcrypt.compatibility@gmail.com
    android.database.sqlite.SQLiteConstraintException: UNIQUE constraint failed: attachment.email, attachment.uid, attachment.folder, attachment.attachment_id (code 2067)
        at android.database.sqlite.SQLiteConnection.nativeExecuteForLastInsertedRowId(Native Method)
        at android.database.sqlite.SQLiteConnection.executeForLastInsertedRowId(SQLiteConnection.java:784)
        at android.database.sqlite.SQLiteSession.executeForLastInsertedRowId(SQLiteSession.java:788)
        at android.database.sqlite.SQLiteStatement.executeInsert(SQLiteStatement.java:86)
        at android.database.sqlite.SQLiteDatabase.insertWithOnConflict(SQLiteDatabase.java:1474)
        at android.database.sqlite.SQLiteDatabase.insert(SQLiteDatabase.java:1343)
        at com.flowcrypt.email.database.provider.SecurityContentProvider.bulkInsert(SecurityContentProvider.kt:142)
        at android.content.ContentProvider$Transport.bulkInsert(ContentProvider.java:282)
        at android.content.ContentResolver.bulkInsert(ContentResolver.java:1604)
        at com.flowcrypt.email.database.dao.source.imap.AttachmentDaoSource.addRows(AttachmentDaoSource.kt:102)
        at com.flowcrypt.email.service.EmailSyncService.updateAttTable(EmailSyncService.kt:582)
        at com.flowcrypt.email.service.EmailSyncService.onMsgDetailsReceived(EmailSyncService.kt:238)
        at com.flowcrypt.email.api.email.sync.tasks.LoadMessageDetailsSyncTask.runIMAPAction(LoadMessageDetailsSyncTask.kt:79)
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.runSyncTask$FlowCrypt_devDebug(EmailSyncManager.kt:519)
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$ActiveSyncTaskRunnable.run(EmailSyncManager.kt:614)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:457)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
        at java.lang.Thread.run(Thread.java:764)
D/SecurityContentProvider: Failed to insert row into content://com.flowcrypt.email.debug.SecurityContentProvider/attachment
I/System.out: B21 UNSELECT
I/System.out: B21 OK Returned to authenticated state. (Success)
I/System.out: DEBUG GIMAPS: added an Authenticated connection -- size: 1
D/ActiveSyncTaskRunnable: The task = LoadMessageDetailsSyncTask completed
    ActiveSyncTaskBlockingQueue size = 0
```

**Comment by tomholub** [comment]
Keep getting this every time I open the same message

**Comment by DenBond7** [comment]
@tomholub Are you able to reproduce it again? Do you have a freshly installed app or updated? (if updated, Do you remember what was the original number of the app version?)

**Comment by tomholub** [comment]
It was the 0.8.1 or 0.8.2 branch that I was developing, you can look at the last commit at the time I posted it and that's the exact commit I was working with.

I don't have time to try to reproduce it now but I can try a few days later.

**Comment by tomholub** [comment]
I don't know which exact message, but it's on compatibility account, and I see from the log that it has attachment id `f_UgPXIpPpLq`

**Comment by DenBond7** [comment]
@tomholub 

>I don't know which exact message, but it's on compatibility account, and I see from the log that it has attachment id f_UgPXIpPpLq

I know the message

https://mail.google.com/mail/u/2/#search/has%3Aattachment+0x0D5688EBF3102BE7.asc.pgp/WhctKHTKmxpLDdNZTPXjPqdDDNwfNtjsNnLWQnHdcwLbBwLSmWsWrncJGzPRtVNHVqrLBHV

**Comment by tomholub** [comment]
Good. I can try to reproduce it later

**Comment by DenBond7** [comment]
fixed via #343
